src/SqlVirtualMachine/examples/New-AzSqlVM.md

```diff
@@ -20,3 +22,87 @@ Location	Name		ResourceGroupName
 eastus		sqlvm1		ResourceGroup01	
 ```
 
+Create a SQL virtual machine with specific sku type and license type.
```

Comment:
Please run "build-module.ps1" after updating examples. It'll apply the changes in examples to docs. Commit the updated  docs too.

Reply:
Updated docs folder.